config/internal/apiserver/default/deployment.yaml.tmpl

```diff
@@ -120,10 +120,10 @@ spec:
             - name: SIGNED_URL_EXPIRY_TIME_SECONDS
               value: "{{.APIServer.ArtifactSignedURLExpirySeconds}}"
             {{ if .PodToPodTLS }}
-            - name: ML_PIPELINE_TLS_ENABLED
```

Comment:
Could you please remove ARTIFACT_COPY_STEP_CABUNDLE_MOUNTPATH ,ARTIFACT_COPY_STEP_CABUNDLE_CONFIGMAP_KEY and ARTIFACT_COPY_STEP_CABUNDLE_CONFIGMAP_NAME if not needed anymore.